identity/src/main/java/com/stripe/android/identity/ui/DocumenetScanScreen.kt

```diff
@@ -0,0 +1,163 @@
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.semantics.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.viewinterop.AndroidView
+import com.google.android.material.composethemeadapter.MdcTheme
+import com.stripe.android.camera.scanui.CameraView
+import com.stripe.android.identity.R
+import com.stripe.android.identity.states.IdentityScanState
+
+internal const val CONTINUE_BUTTON_TAG = "Continue"
+internal const val SCAN_TITLE_TAG = "Title"
+internal const val SCAN_MESSAGE_TAG = "Message"
+internal const val CHECK_MARK_TAG = "CheckMark"
+internal const val VIEW_FINDER_ASPECT_RATIO = 1.5f
+
+@Composable
+internal fun DocumentScanScreen(
+    title: String,
+    message: String,
+    newDisplayState: IdentityScanState?,
+    onCameraViewCreated: (CameraView) -> Unit,
+    onContinueClicked: () -> Unit
+) {
+    MdcTheme {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(
+                    vertical = dimensionResource(id = R.dimen.page_vertical_margin),
+                    horizontal = dimensionResource(id = R.dimen.page_horizontal_margin)
+                )
+        ) {
+            Column(
+                modifier = Modifier
+                    .weight(1f)
+                    .verticalScroll(rememberScrollState())
+            ) {
+                Text(
+                    text = title,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .semantics {
+                            testTag = SCAN_TITLE_TAG
+                        },
+                    fontSize = 24.sp,
+                    fontWeight = FontWeight.Bold
+                )
+                Text(
+                    text = message,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(100.dp)
+                        .padding(
+                            top = dimensionResource(id = R.dimen.item_vertical_margin),
+                            bottom = 48.dp
+                        )
+                        .semantics {
+                            testTag = SCAN_MESSAGE_TAG
+                        },
+                    maxLines = 3
```

Comment:
Maybe it defaults to it, but should we ellipsize when the content goes beyond 3 lines?